public/projects/international-accessibility-task-force/iatf-bot.mdx

```diff
@@ -0,0 +1,141 @@
+---
+name: International Accessibility Task Force [discord bot]
+repoUrl: https://github.com/international-accessibility-task-force/iatf-bot
+description: International Accessibility Task Force is an open-to-everyone community of assistive technology users, open-source developers, and people interested in accessibility and inclusion.
+languages: # A list of programming languages used in this project
+  - JavaScript
+  - NodeJS
+  - DiscordJS
+  - HTML / CSS
+  - Markdown
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Social Activism
+  - Accessibility
+  - Community
+  - Open-Source
+  - Discord
+  - Bot
+  - npm
+  - lint
+  - eslint
+  - babel
+  - a11y
```

Comment:
**chore:** let's stick with the existing tags, although adding "Accessibility" is a great idea so let's keep that one.
```suggestion
```